hello-impl/src/main/resources/application-prod.conf

```diff
@@ -25,16 +25,4 @@ akka.management {
       service-name = "hello-lagom-java-openshift-smoketests-service"
     }
   }
-
-  http {
-    port = 8558
-    bind-hostname = "0.0.0.0"
-    route-providers += "akka.management.HealthCheckRoutes"
-  }
-
-  health-checks {
-    readiness-checks {
-      cluster-membership = "akka.management.cluster.scaladsl.ClusterMembershipCheck"
-    }
-  }
 }
```

Comment:
These settings are no longer needed. This is the only valuable change in the PR.